kie-wb/kie-wb-webapp/pom.xml

```diff
@@ -1504,7 +1502,7 @@
             <server>org.jboss.errai.cdi.server.gwt.EmbeddedWildFlyLauncher</server>
             <disableCastChecking>true</disableCastChecking>
             <runTarget>kie-wb.html</runTarget>
-            <hostedWebapp>src/main/webapp</hostedWebapp>
+            <hostedWebapp>${project.build.directory}/kie-wb.war</hostedWebapp>
```

Comment:
Would it be now possible to fix the output directory directly here https://github.com/mswiderski/kie-wb-distributions/blob/hosted-mode-executor-fix/kie-wb/kie-wb-webapp/pom.xml#L1475 instead of copying stuff around (e.g. those antrun executions)?